plotly/matplotlylib/mpltools.py

```diff
@@ -453,13 +453,13 @@ def prep_ticks(ax, index, ax_type, props):
         else:
             axis_dict['tick0'] = tick0
             axis_dict['dtick'] = dtick
-            axis_dict['tickmode'] = False
+            axis_dict['tickmode'] = None
     elif scale == 'log':
         try:
             axis_dict['tick0'] = props['axes'][index]['tickvalues'][0]
             axis_dict['dtick'] = props['axes'][index]['tickvalues'][1] - \
                             props['axes'][index]['tickvalues'][0]
-            axis_dict['tickmode'] = False
+            axis_dict['tickmode'] = None
```

Comment:
Seems like this should be tick mode of `'log'`.  Would you mind trying this out on a simple matplotlib figure with a log axis?

Reply:
This is a [matplotlib log example](https://matplotlib.org/examples/pylab_examples/log_demo.html):
![image](https://user-images.githubusercontent.com/18031651/43848112-89c50f30-9b32-11e8-800d-d7f1a175c6ec.png)
**With `None`, it looks like:**
![image](https://user-images.githubusercontent.com/18031651/43848149-a3b86cd4-9b32-11e8-9a8a-f5b41b74cbae.png)

and brings the warnings:
```
plotly.graph_objs.Annotation is deprecated. 
...
Converted non-base10 x-axis log scale to 'linear'
...
Dang! That path collection is out of this world. I totally don't know what to do with it yet! Plotly can only import path collections linked to 'data' coordinates
```
**With `log`, it looks like:**
![image](https://user-images.githubusercontent.com/18031651/43848333-000108d4-9b33-11e8-88e7-76f63f095069.png)
Same warning as in the `None` setting here.
I cannot see a difference between `None` and setting to `log`. What should it then be?
Despite that, there are obviously troubles converting non base10 logs.